_posts/2015-12-03-journey_to_high_availability.md

```diff
@@ -8,6 +8,8 @@ tags:
 - puppet
 - high_availability
 guest_post: true
+guest_url:
+https://chrispisano.wordpress.com/2016/01/18/journey-to-high-availability-2/
```

Comment:
ah, must be my 80 character/line limit for .md files in vim. . . i can fix it today